docs/openapi.yaml

```diff
@@ -319,12 +335,14 @@ paths:
 
   /contract/{contractTxId}/{field}:
     get:
-      summary: Returns the field in the contract state for a given transaction id
-      description: Returns the field in the contract state for a given transaction id
+      deprecated: true
+      summary: Returns the field in the contract state for a given transaction id. For traversing contract state, use /contract/{contractTxId}/state/{nestedPath}
+      description: Returns the field in the contract state for a given transaction id. For traversing contract state, use /contract/{contractTxId}/state/{nestedPath}
```

Comment:
just marking our existing one layer deep handler as deprecated in favor of the new one